project/package.mill.scala

```diff
@@ -0,0 +1 @@
+package build.project
```

Comment:
Why do we need this? Maybe just add a comment

Reply:
It's how [multi file builds](https://mill-build.org/mill/large/multi-file-builds.html) work in Mill.

Reply:
Mill 0.12 syntax.